sdks/python/container/boot.go

```diff
@@ -73,13 +74,20 @@ const (
 )
 
 func main() {
+	if err := mainError(); err != nil {
+		log.Print(err)
+		os.Exit(1)
```

Comment:
This is already what happens when log.Fatal is being called, so it's unclear why this whole change was made?

https://pkg.go.dev/log#Fatal

Reply:
I answered this somewhere above. log.Fatal prevents clean-up of resources, so we cannot shut down the Python worker process cleanly. Without this change, you will end up producing a lot of un-wait()-ed zombie processes and potentially leave temporary files in place etc.

Reply:
OK, that's fair. The question is now why not use *log.Fatal* here instead of the manual print + Exit. We're already out of the function scope.

Reply:
I just don't like implicit logic like that. Maybe that's because I'm not a Go programmer, but I was totally surprised to see a simple log print call to hard-exit the main process. It is totally non-obvious and you have to read the documentation to understand what's happening. This is not something one would expect unless perhaps you are used to Go idiosyncrasies. I changed it to log.Fatal() since you seem to prefer it.

Reply:
In this case the logic isn't implicit for log.Fatal and log.Fatalf. It's both the documented and implemented behavior. It's a signal to the reader that this is an expected errorful exit.

Go as a rule is fairly explicit, but it also needs to be balanced with avoiding verbose repetition. Hence functions to improve readability. This allows readers to note that something interesting is going on, vs the split call, when they see an explicit Print followed by an exit.